website/docs/docs/cloud/git/git-configuration-in-dbt-cloud.md

```diff
@@ -1,37 +1,46 @@
 ---
-title: "Git configuration in dbt Cloud"
+title: "Configure Git in dbt Cloud"
 description: "Learn about the Git providers supported in dbt Cloud"
-pagination_next: "docs/cloud/git/import-a-project-by-git-url"
-pagination_prev: null
+hide_table_of_contents: true
+pagination_next: "docs/cloud/git/managed-repository"
 ---
 
-<div className="grid--2-col">
+[Version control](/docs/collaborate/git/version-control-basics) is an essential part of the dbt workflow, enabling teams to collaborate effectively and maintain a history of changes to their dbt projects. 
```

Comment:
Not a blocker here, but in the future,e we might want to open with a little blurb about what Git is